### PR TITLE
Coverage

### LINKED ISSUE #222: Add code coverage badge
@natct10

### PR DESCRIPTION
**Pull request by natct10**
- [x] Generation of a `.coverage` file with `pytest-cov` and update `gitignore`.
- [x] Access token from SubstraFoundation organisation for https://codecov.io (only for this repo and for the master branch)
- [x] Add token to Travis & update test command
- [x] Add the tests commands in the README 

Close #222

### CODE REVIEW

**Review by bowni — COMMENTED** [review]
Looks awesome already!

File: mplc/doc/documentation.md [hunk 1]
```diff
@@ -544,6 +544,23 @@ These are not mandatory, by default the local dataset will not be split.
...
+# or 
+python3 -m pytest -vv tests/unit_tests.py
+
+# Fall in debugger breakpoint in case of error
+pytest -vv tests/unit_tests.py --pdb
+
+# Run tests and produce a coverage measure. This will output a .coverage file
+pytest -vv --cov=mplc tests/unit_tests.py
+```
+
```
Comment (arthurPignet) [inline mplc/doc/documentation.md:563]: I am not sure, why don't put these explanations in the CONTRIBUTING.md file ?

**Review by arthurPignet — APPROVED** [review]

File: mplc/doc/documentation.md (hunk 1 above)
Comment (natct10) [reply mplc/doc/documentation.md:563]: You mean removing this from the documentation.md?

Comment (natct10) [reply mplc/doc/documentation.md:563]: I have added this to the contributing doc

**Review by RomainGoussault — APPROVED** [review]
Nice !!

**Pull request merged**